test/util/login-helpers.js

```diff
@@ -52,16 +51,29 @@ function initUserMock(level) {
 }
 /* eslint-enable camelcase */
 
-function logIn(client, url, level) {
-  initUserMock(level);
+let tokenCounter = 0;
+
+function getUserToken() {
+  return `token-${process.pid}-${tokenCounter++}`;
+}
+
+function logIn(token, client, url, level) {
+  initUserMock(token, level);
+
   client
     .url(`${E2eHelpers.baseUrl}${url}`)
     .waitForElementVisible('body', Timeouts.normal);
```

Comment:
I _believe_ that's the case. I'll set up to do some more testing in the future if we find there's a good performance improvement to be had from this.